src/action/count.rs

```diff
@@ -1,13 +1,18 @@
-use bson::Document;
+use bson::{Bson, Document};
+use mongodb_internal_macros::{option_setters_2, options_doc};
+use std::time::Duration;
 
 use crate::{
-    coll::options::{CountOptions, EstimatedDocumentCountOptions},
+    coll::options::{CountOptions, EstimatedDocumentCountOptions, Hint},
```

Comment:
This is kind of an annoying consequence - because the setters are taken from the options struct as a token stream, the types have to be re-imported here.  Actual compile-time reflection with type information rather than just textual macros would be the answer but that got dramabombed so who knows if rust will ever get that :(